cl/opinion_page/views.py

```diff
@@ -479,6 +527,87 @@ def view_recap_document(
                 rd.docket_entry.docket.court_id, "US/Eastern"
             ),
             "redirect_to_pacer_modal": redirect_to_pacer_modal,
+            "authorities_context": authorities_context,
+        },
+    )
+
+
+def view_recap_authorities(
+    request: HttpRequest,
+    docket_id: int | None = None,
+    doc_num: int | None = None,
+    att_num: int | None = None,
+    slug: str = "",
+    is_og_bot: bool = False,
+) -> HttpResponse:
+    """This view can display authorities of an attachment or a regular
+    document, depending on the URL pattern that is matched.
+    """
+    rd = RECAPDocument.objects.filter(
+        docket_entry__docket__id=docket_id,
+        document_number=doc_num,
+        attachment_number=att_num,
+    ).order_by("pk")[0]
+    title = make_rd_title(rd)
+    rd = make_thumb_if_needed(request, rd)
+
+    try:
+        note = Note.objects.get(recap_doc_id=rd.pk, user=request.user)
+    except (ObjectDoesNotExist, TypeError):
+        # Not saved in notes or anonymous user
+        note_form = NoteForm(
+            initial={
+                "recap_doc_id": rd.pk,
+                "name": trunc(title, 100, ellipsis="..."),
+            }
+        )
+    else:
+        note_form = NoteForm(instance=note)
+
+    if rd.attachment_number:
+        view_authorities_url = reverse(
+            "view_attachment_authorities",
+            args=[
+                rd.docket_entry.docket_id,
+                rd.document_number,
+                rd.attachment_number,
+                rd.docket_entry.docket.slug,
+            ],
+        )
+    else:
+        view_authorities_url = reverse(
+            "view_document_authorities",
+            args=[
+                rd.docket_entry.docket_id,
+                rd.document_number,
+                rd.docket_entry.docket.slug,
+            ],
+        )
```

Comment:
I'd suggest moving this to the template too, but if you think it's better here, let's definitely not repeat it. DRY!